tests/fixture/tmpnet/node.go

```diff
@@ -53,7 +53,12 @@ type NodeRuntime interface {
 
 // Configuration required to configure a node runtime.
 type NodeRuntimeConfig struct {
+	Process *ProcessRuntimeConfig
+}
```

Comment:
So the aim is to then have a Process and Kube field right?
Could/Should we instead have an interface here, with the process implementation and a future kube implementation? 🤔 
Maybe I'm missing the point completely though, in which case, my apologies 😉